docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-issues/kubernetes.md

```diff
@@ -0,0 +1,309 @@
+Warning  FailedScheduling   75s (x40 over 55m)    default-scheduler   0/55 nodes are available: 19 Insufficient cpu, 36 node(s) didn't match Pod's node affinity/selector
+```
+
+**Resolution**
+
+We can schedule YBA pods on the particular nodes as required. We might see pods in the Pending state if the scheduler fails to figure out a node because pods need to match the labels or taints.
+- For [Node Selector](https://docs.yugabyte.com/stable/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes/#nodeselector)
+- For [Affinity](https://docs.yugabyte.com/stable/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes/#zoneaffinity)
+- For [Taints & Tolerations](https://docs.yugabyte.com/stable/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes/#toleration)
+
+### Is StorageClass VolumeBindingMode set to WaitForFirstConsumer?
+
+**Symptom**
+
+You will see some pods pending during the multi-zone deployment of YugabyteDB. If we do not set the `VolumeBindingMode` to `WaitForFirstConsumer`, the cluster might create volume in a different zone than the selected one.
+
+**Debug**
+
+You can use following commands to check the storage class related information.
+
+- Get all storage classes along with their `VolumeBindingMode` in the cluster.
+
+  ```sh
+  kubectl get storageclass
+
+  ## output
+  ## NAME                 PROVISIONER             RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+  ## premium-rwo          pd.csi.storage.gke.io   Delete          WaitForFirstConsumer   true                   418d
+  ## standard (default)   kubernetes.io/gce-pd    Delete          Immediate              true                   418d
+  ## standard-rwo         pd.csi.storage.gke.io   Delete          WaitForFirstConsumer   true                   418d
+  ```
+
+- Get information for particular storage class
+
+  ```sh
+  ## Considering standard named storage class exists in the cluster.
+  kubectl describe storageclass standard
+
+  ## output
+  ## Name:                  standard
+  ## IsDefaultClass:        Yes
+  ## Annotations:           storageclass.kubernetes.io/is-default-class=true
+  ## Provisioner:           kubernetes.io/gce-pd
+  ## Parameters:            type=pd-standard
+  ## AllowVolumeExpansion:  True
+  ## MountOptions:          <none>
+  ## ReclaimPolicy:         Delete
+  ## VolumeBindingMode:     Immediate
+  ## Events:                <none>
+  ```
+
+**Resolution**
+
+Ensure the `StorageClass` used during YBA deployment should have `WaitForFirstConsumer` as `VolumeBindingMode`. Use the following command snippet to update the `VolumeBindingMode` of a `StorageClass`.
+
+  ```sh
+  ## Considering standard named storage class exists in the cluster.
+  kubectl get storageclass standard -ojson | jq '.volumeBindingMode="WaitForFirstConsumer" | del(.metadata.managedFields, .metadata.creationTimestamp, .metadata.resourceVersion, .metadata.uid)' | kubectl replace --force -f -
+  ```
+
```

Comment:
It would be good to have a high level section similar to the first one ("unable to schedule pod").

The overall structure could be

## Pods wasn't scheduled
### Resources
### affinity ...
## Pod was scheduled but is not running
### image pull secret
### crash loop back off
## Pod is running but load balancer service is not ready
### EKS controller
### load balancer in pending state
## Other issues
### CORS
...